server/controllers/request.js

```diff
@@ -0,0 +1,87 @@
+const Request = require("../models/Request");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /request
+// @desc add request
+// @access Public
+exports.addRequest = asyncHandler(async (req, res, next) => {
+  const { userID, sitterID, startDate, endDate, accept, decline, paid } =

```

Comment:
Since we are creating a request here, accept, decline and paid are going to be false at first. You can exclude them here and add a default value of false in the Request Model for each of these fields maybe?